privaterelay/templates/includes/modal-delete.html

```diff
@@ -10,7 +10,8 @@
     <div class="c-modal-body">
       <span class="alias-to-delete js-alias-to-delete demi"></span>
       <p class="modal-message">{% ftlmsg 'modal-delete-warning-recovery-html' email='' %}</p> 
-      <p class="warning-text  modal-message">{% ftlmsg 'modal-delete-warning-upgrade' %}</p>
+      <p class="warning-text  modal-message js-relay-address-warning">{% ftlmsg 'modal-delete-warning-upgrade' %}</p>
+      <p class="warning-text  modal-message hidden js-domain-address-warning">{% ftlmsg 'modal-delete-domain-address-warning-upgrade' %}</p>
```

Comment:
Can you  please revise this class to be `is-hidden`? The goal is for [state based classes](https://github.com/mozilla/fx-private-relay/blob/main/docs/coding-standards.md?plain=1#L23) to start with `is-`.  

```suggestion
      <p class="warning-text  modal-message is-hidden js-domain-address-warning">{% ftlmsg 'modal-delete-domain-address-warning-upgrade' %}</p>
```